packages/next/src/build/index.ts

```diff
@@ -3104,13 +3104,11 @@ export default async function build(
           .traceFn(() => printCustomRoutes({ redirects, rewrites, headers }))
       }
 
+      // TODO: remove in the next major version
       if (config.analyticsId) {
-        console.log(
-          bold(green('Next.js Speed Insights')) +
-            ' is enabled for this production build. ' +
-            "You'll receive a Real Experience Score computed by all of your visitors."
+        Log.warnOnce(
```

Comment:
```suggestion
        Log.warn(
```

`Log.warnOnce` is only required if it runs multiple times, this code should already only runs once.
Saves some extra memory being assigned as these messages are kept in a `Set`.